<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_home_window</class>
 <widget class="QMainWindow" name="customer_home_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>212</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>635</width>
    <height>212</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Portale Tipografia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_titolo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Benvenuto, &lt;utente&gt;!</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_user_email">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;email_utente&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stampa_doc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stampa un documento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_market">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Fai un acquisto online</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
